htdocs/tools/css.php

```diff
@@ -171,50 +171,3 @@ function csr_css_loadfile($file,$location=''){
 	$css_file->createRelativePath();
 	return $css_file->load($location);
 }
-
-
-
-/**
- * Load style ini contents
- *
- * @author Andreas Gohr <andi@splitbrain.org>
- * @author Gerrit Uitslag <klapinklapin@gmail.com>
- *
- * @param string $layout the used layout
- * @return array with keys 'stylesheets' and 'replacements'
- */
-function css_csr_styleini($layout) {
-	$stylesheets = array(); // mode, file => base
-	$replacements = array(); // placeholder => value
-
-	// load template's style.ini
-	$incbase = HTDOCS_PATH;
-	$webbase = CSR_ROOT;
-	$ini = $incbase . $layout . '/style.ini';
-	if (file_exists($ini)) {
-		$data = parse_ini_file($ini, true);
-
-		// stylesheets
-		if (is_array($data['stylesheets'])) foreach ($data['stylesheets'] as $module => $files) {
-			foreach ($files as $file) {
-				if(DEBUG && substr($file, -8) == '.min.css') {
```

Comment:
Dit zorgt ervoor dat je in DEBUG modus non-minified stylesheet files krijgt. Erg fijn voor debuggen uiteraard.

Weet je zeker dat dit niet gebruikt wordt (lokaal / in productie)?